utils/get-soap-client.js

```diff
@@ -11,10 +11,10 @@ var memoize          = require('memoizee')
 
   , createSoapClient = promisify(soap.createClient);
 
-var getSoapClientImpl = memoize(function (wsdlUrl, soapUrl) {
-	debug('creating soap client (%s)', soapUrl);
+var getSoapClientImpl = memoize(function (wsdlUrl, options) {
```

Comment:
As the name suggests, `options` should be optional, we don't expose them in the signature of functions, other than in comment.